moto/iam/models.py

```diff
@@ -733,6 +733,10 @@ def create_policy(self, description, path, policy_document, policy_name):
         policy = ManagedPolicy(
             policy_name, description=description, document=policy_document, path=path
         )
+        if policy.arn in self.managed_policies:
+            raise EntityAlreadyExists(
+                f"A policy called {policy_name} already exists. Duplicate names are not allowed."
```

Comment:
f-strings were introduced in Python 3.6 for the first time and moto also supports Python 2.7. Please change this line to
`"A policy called {} already exists. Duplicate names are not allowed.".format(policy_name)`

Reply:
Fixed!